%!TEX TS-program = xelatex
\documentclass{nihbiosketch}

%\usepackage{draftwatermark}  % delete this in your document!
%\SetWatermarkText{Sample}    % delete this in your document!
%\SetWatermarkLightness{0.9}  % delete this in your document!

%------------------------------------------------------------------------------
\name{Brunson, Jason Cory}
\eracommons{brunsonj}
\position{Postdoctoral Fellow at the Center for Quantitative Medicine}

\begin{document}
%------------------------------------------------------------------------------

\begin{education}
Virginia Tech, Blacksburg & B.S. & 08/2000 & 05/2004 & Mathematics \\
Virginia Tech, Blacksburg & B.S. & 08/2001 & 08/2004 & Statistics \\
Virginia Tech, Blacksburg & M.S. & 08/2004 & 05/2005 & Mathematics \\
Virginia Tech, Blacksburg & Ph.D. & 08/2005 & 12/2013 & Mathematics \\
UConn Health, Farmington & Postdoc & 06/2014 & present & Biomedical Informatics \\
\end{education}

%------------------------------------------------------------------------------
\section{Personal Statement}

\begin{statement}

I am ideally situated to conduct the proposed project to develop statistical topological methods for healthcare data analysis.
My PhD training in algebraic geometry included training in homological algebra and graph theory, and I have contributed to the theory, review, validation, and application of network science (covered in ``Contribution to Science'' below). Additionally, my sponsor has a strong record of research in the related field of combinatorial topology.
At my current appointment I have trained in data exploration and statistical inference, including dimension reduction, feature extraction, hierarchical regression, and Bayesian estimation (``Collaborations'' and ``Contribution to Science''). These are among the foundational concepts for topological data analysis (TDA), and as part of an ongoing postdoc collaboration I have written an original implementation of popular TDA tools.
I have also familiarized myself with the manipulation and exploration of diverse sources of healthcare data during my appointment, including an EHR database and both curated and raw insurance claims. Much of my proposed training is in conventional clinical study designs and healthcare analytics, for which I am prepared both to learn best practices and to interrogate underlying assumptions.
I have a strong record of professional organization, as a bargaining unit representative and a postdoctoral association officer as well as event organizer and seminar contributor, which will be important to coordinating my training and to the more intensive professional networking ahead.
Finally, I have mentored several research projects with undergraduate and high school students, which will be essential to producing a scientific work force comfortable and deft with TDA in my next role as an independent researcher.

I've acquired a range of subject matter experience by way of multiple shifts in my research agenda and career goals. I began my career in network science concurrently with my PhD program in algebraic geometry, and my current work in health informatics draws upon statistical training and programming experience developed for scientometrics.
While I trained as a pure mathematician, my work on real-world applications has been most rewarding, for both the broad utility of the methods developed and the societal significance of the results obtained. Some specific instances are the importance of trends in collaboration incentives to the conversation about publishing ethics and of computer literacy to the efficient and timely conduct of pharmacosurveillance. At the same time, the most exciting part of applications research is often recognizing important mathematical connections, including my generalized conceptions of triad census and triad closure and my operationalization of alluvial diagrams. I see the potential for this project to contribute to mathematical theory, medical knowledge, and methods accessibility, and as such I view it as the foundation for a long-term research program.
%I view this project as the foundation for a long-term research program that will contribute to mathematical theory, medical knowledge, and accessible software.

\end{statement}

%------------------------------------------------------------------------------
\section{Positions and Honors}

\subsection*{Positions and Employment}
\begin{datetbl}
2010--2013 & Visiting Research Assistant, Virginia Bioinformatics Institute, Virginia Tech, Blacksburg, VA \\
2014       & Adjunct Professor, Department of Mathematics, Radford University, Radford, VA \\
2014--     & Postdoctoral Fellow, Center for Quantitative Medicine, UConn Health, Farmington, CT \\
\end{datetbl}

\subsection*{Other Experience and Professional Memberships}
\begin{datetbl}
2014--     & Member, Society for Industrial and Applied Mathematics (SIAM) \\
2015--     & Founding member, UConn Health--JAX-GM Postdoctoral Association (UJPDA) \\
2017--2018 & President, UJPDA \\
2018--     & Member, American Medical Informatics Association (AMIA) \\
\end{datetbl}

\subsection*{Collaborations}
\begin{datetbl}
2015--2016 & ``Analysis of increased compound drug prescriptions in Connecticut 2014--2015'', training project with S. Czunas and T. Woodruff for the Office of the State Comptroller of Connecticut \\
2016--2017 & ``Identifying molecular pathways that lead to CD8+ T cell memory formation'', collaboration with T. Samji in Departments of Immunology at UConn Health and New York University \\
%2017--     & ``Topological analysis of single-cell gene expression data'', collaboration with A. Konstorum at CQM \\
%2017--     & ``Energy values of minimal functional routes'', collaboration with L.S. Vieira at CQM \\
%2018       & ``Developing a User-Friendly R Package Providing Standardized Coding and Analytic Methods for Comparative Effectiveness Research Using Administrative Healthcare Claims Data'', consultancy with C. Coleman at the UConn School of Pharmacy and Hartford Hospital \\
\end{datetbl}

\subsection*{Professional Service}
\begin{datelngtbl}
2010       & Mentor, Research Experience for Undergraduates (REU) on Modeling and Simulation in Systems Biology \\
2012--2014 & Co-Organizer, Virginia Tech Grad Student Speed Dating \\
2015       & Co-PI, ACSB 2015: A Conference on Algebraic and Combinatorial Approaches in Systems Biology, (NSF DMS-1503562, PI Vera--Licona) \\
2015--2017 & Postdoc Representative and Negotiating Team Member, University Health Professionals AFT Local 3837 (UHP) \\
2015--     & Front Desk (clinical data entry, visit coordination), Hartford Gay and Lesbian Health Collective \\
2016--     & Reviewer, AMIA \\
2016, 2018 & Mentor, High School Student Research Apprentice Program (2018 scheduled) \\
2017       & Poster judge, Medical and Dental Student Research Day \\
2017       & Co-organizer, Postdoc Research Day \\
2017       & Mentor, REU on Modeling and Simulation in Systems Biology \\
2017       & Drop-in editing, Tool Kit for Scientific Communication course \\
2018       & Co-organizer, UConn Health Speed Networking \\
\end{datelngtbl}

\subsection*{Presentations}
\begin{datelngtbl}
2012       & ``Evolution of the mathematics research collaboration network'': Graduate Student Association Research Symposium, Virginia Tech \\
2012       & Lecture series on Schubert calculus, Virginia Bioinformatics Institute \\
2013       & ``Caution in interpreting graph-theoretic diagnostics'', SIAM Student Seminar \\
2014       & ``Surveying the Diagnostic Landscape'', Mining Networks and Graphs: A Big Data Analytic Challenge, SIAM International Conference on Data Mining \\
2014       & ``Triad census for two-mode networks'', SIAM Workshop on Network Science \\
2014       & ``Evolving Collaboration Patterns in Medical Research'', AMIA Annual Symposium \\
2015       & Tutorial on data analysis and visualization in R, Postdoctoral Seminar, UConn Health \\
2016       & ``Scientometrics in Space and Time'', CQM Faculty/Staff seminar \\
2016       & ``Modulus on graphs as a generalization of standard graph theoretic quantities'', Mathematics in Medicine (MiM) Journal Club \\
2016       & ``Emerging network methods in healthcare informatics'', Connecticut Institute for Clinical and Translational Science (CICATS) methods seminar \\
2017       & ``RPKM versus TPM for comparing multiple gene expression across multiple RNA-seq samples'', Laubenbacher Group meeting \\
2017       & ``Power-law distributions in empirical data'', MiM Journal Club + CICATS methods seminar \\
2017       & ``Applications of network analysis to routinely collected health care data'', CQM presentation, UConn Health \\
2017       & ``Modeling Incidence and Severity of Disease using Administrative Healthcare Data'', Open Data in Action, Hartford Public Library \\
2018       & ``Co-occurrence Networks from Correlation Matrices'', CICATS methods seminar \\
2018       & ``Conventional versus topological data analysis for disease subtyping: cases of type-2 diabetes mellitus'', MiM Journal Club (scheduled) \\
2018       & ``Pairwise versus multivariate constructions of co-occurrence networks'', SIAM Workshop on Network Science (accepted) \\
2018       & ``Network analysis to measure disparities in professional healthcare infrastructure'', MiM Journal Club (scheduled) \\
\end{datelngtbl}

%\subsection*{Honors}
%\begin{datetbl}
%2017       & Union Representative of the Year, UHP \\
%\end{datetbl}

%------------------------------------------------------------------------------
\section{Contribution to Science}

\begin{enumerate}

\item \emph{Scientometrics.}
Together with a team of four undergraduates, I was introduced to network analysis and scientometrics at a summer program organized around systems biology. We conducted a thorough analysis of the network and took an original approach to mapping its evolution over time, using a custom family of nonlinear changepoint models to characterize two pronounced shifts in coauthorship structure. Our study has been cited several times since, often as a touchstone for patterns of collaboration in one discipline (mathematics) as contrasted with others, though also as a launch point for theoretical work. Some follow-up analysis led to the design of a study on coauthorship rates in the much larger biomedical literature, in which we took the first step toward teasing apart the entangled contributions of methodological complexity and research culture on the accelerating rise of multiple authorship. From here, several directions remain to be explored, including a careful validation of subject classification assignments as a proxy for disciplinary scope and a range of proposed network-based metrics of researcher specialization and diversification.

\begin{enumerate}
\item J.C. Brunson, S. Fassino, A. McInnes, M. Narayan, B. Richardson, C. Frank, P. Ion, and R.C. Laubenbacher, Evolutionary events in a mathematical sciences research collaboration network, \emph{Scientometrics} 99(3): 973--998, 2014.
\item J.C. Brunson, X. Wang, and R.C. Laubenbacher, Effects of research complexity and competition on the incidence and growth of coauthorship in biomedicine, \emph{PLoS One} 12(3): e0173444, 2017.
\end{enumerate}

\item \emph{Network science.}
Our analysis of mathematics research output prompted a follow-up question that turned out to be remarkably under-studied: How can triadic closure be understood in an affiliation (bipartite) network context? My efforts to resolve this problem led to the unexpected discovery of a family of graph statistics, parameterized in category-theoretic terms, that included the classical clustering coefficient (evaluated on the unipartite projection) and both families of bipartite statistics that had been proposed up to that point.
More recently, I've had published a systematic review of studies that employ network analysis techniques for secondary use of healthcare data. This literature contains myriad projects in diverse domains that in several respects stand to benefit from sharing and combining techniques, and we brought them together into a single reference (and methodological synthesis) we hope will improve collaborations and accelerate advances.
One lesson from this review was that the increasingly popular use of ``comorbidity networks'' suffers from inconsistent methodology and uncertain validity. In response, we're just completing a sensitivity analysis of the comorbidity network construction and several network statistics that have been used to characterize them, which we hope will encourage more consistent and transparent use of this concept in the future.

\begin{enumerate}
\item J.C. Brunson, Triadic analysis of affiliation networks, \emph{Network Science} 3(4): 480--508, 2015.
\item J.C. Brunson and R.C. Laubenbacher, Applications of network analysis to routinely collected healthcare data: a systematic review, \emph{Journal of the American Medical Informatics Association} 25(2): 210--221, 2018.
\item J.C. Brunson, T.P. Agresta, and R.C. Laubenbacher, Reproducibility and sensitivity of comorbidity network analysis, in preparation.
\end{enumerate}

\item \emph{Healthcare analytics.}
My appointment at the Center for Quantitative Medicine entailed self-training as a ``data scientist''---a combination of statistical literacy, computational programming, and consultancy. One project in this role was a training collaboration with the Office of the State Comptroller of Connecticut, in which I combined pharmacy and insurance claims data provided by the state with publicly available healthcare survey data to describe how several out-of-state compound pharmacies pushed product to patients at exorbitant costs to the state. This project pushed me to adopt reproducible documentation as a research standard, which also proved useful for updating reports as new data became available, and some of the visualization software I wrote to communicate my results has been used by several other researchers.
%More recently, I've been assisting an RNA sequencing analysis pipeline and doing auxiliary analyses for a colleague in immunology. Among my contributions are a visual aid for comparing the expression profiles of genes across a common experimental design and some applications of geometric data analysis (GDA) to the identification of next experimental targets. My background in algebraic geometry poises me to develop new tools for GDA, which we expect to introduce in future work.
More recently, I mentored a team of undergraduates in developing risk models for myocardial infarction (MI) using the open-access critical care database MIMIC-III. We built analysis pipelines to contribute to the MIMIC-III code repository and built several MI triage and risk models, which are being written up for submission to a medical journal.
I'm presently engaged in software development with a pharmacist and medical researcher on an R package to expedite comparative effectiveness research using the comprehensive claims database MarketScan. Hundreds of validated specifications for index events, outcome measures, eligibility criteria, and covariates will be incorporated into a simple framework to extract and process the necessary data from standardized tables, substantially reducing the research overhead.

\begin{enumerate}
\item J.C. Brunson, Matrix Schubert varieties for the affine Grassmannian, PhD Thesis with Mark Shimozono, 2013.
\item J.C. Brunson, ggalluvial: Alluvial Diagrams in 'ggplot2'. R package version 0.6.0., 2018, \texttt{\nolinkurl{https://cran.r-project.org/web/packages/ggalluvial/index.html}}.
\item J.C. Brunson, reebr: Statistical Reeb graphs in R, R package, in preparation.
\item J.C. Brunson, cerms: Efficient comparative effectiveness research using MarketScan, R package, in preparation.
\end{enumerate}

\end{enumerate}

\subsubsection*{Complete List of Published Work in MyBibliography:}
%\url{https://orcid.org/0000-0003-3126-9494}
\url{https://www.ncbi.nlm.nih.gov/myncbi/browse/collection/47860258/?sort=date&direction=ascending}

%------------------------------------------------------------------------------
%\section{Research Support}
\section{Additional Information: Research Support and/or Scholastic Performance}

\subsection*{Ongoing Research Support}

\grantinfo{508DE021989-07}{Mina}{07/01/2011--06/30/2021}
{NIDCR}
{Skeletal, Craniofacial, and Oral Biology Training Grant}
{The goal of this proposal is the continuation of the Institutional Training Program in Skeletal, Craniofacial and Oral Biology at the University of Connecticut School of Dental Medicine as a T90/R90 program. It is intended to help meet the substantial need for independent scientists trained in research related to improving oral, dental and craniofacial health in the United States.}
%{Role: PI}

\bigskip

\grantinfo{Internal}{Coleman}{02/01/2018--07/31/2018}
{UConn School of Pharmacy}
{Developing a User-Friendly R Package Providing Standardized Coding and Analytic Methods for Comparative Effectiveness Research Using Administrative Healthcare Claims Data}
{We propose developing a user-friendly R package that will enable the building of a functional research database utilizing standardized covariate and outcomes coding and implement analytic methods for comparative effectiveness research using administrative healthcare claims data.  Having such a package for the ``R Project for Statistical Computing'' platform would both reduce the burden on investigators in preparing administrative claims data for research studies and provide greater accountability and interpretability for end users of these real-world studies (including Bayer AG).}

%------------------------------------------------------------------------------
%\subsection*{Completed Research Support}

%------------------------------------------------------------------------------
\subsection*{Scholastic Performance}

\begin{performance}
2003 & Complex Analysis & 4.00 \\
2003 & Graph Theory & 4.00 \\
2004 & Complex Analysis & 3.30 \\
2004 & Specialized Topics in Algebra (Symmetric Polynomials) & 4.00 \\
2004 & Abstract Algebra & 4.00 \\
2004 & Real Analysis & 3.30 \\
2004 & Combinatorics & 4.00 \\
2005 & Abstract Algebra & 4.00 \\
2005 & Real Analysis & 3.70 \\
2005 & Specialized Topics in Algebra (Elliptic Curves) & 4.00 \\
2005 & Functional Analysis & 3.70 \\
2005 & TS: Lie Groups & 4.00 \\
2005 & TS: Algebraic Topology I & 4.00 \\
2006 & Functional Analysis & 3.00 \\
2006 & TS: Lie Groups & 4.00 \\
2006 & TS: Algebraic Topology II & 3.00 \\
2006 & TS: Introduction to Algebraic Geometry & 4.00 \\
2006 & TS: Several Complex Variables & 4.00 \\
2007 & TS: Several Complex Variables & 4.00 \\
2007 & SS: Mathematics of Computer Simulations & 4.00 \\
2007 & TS: Homological Algebra & 4.00 \\
2007 & Topology and Geometry & 4.00 \\
2008 & TS: Introduction to Algebraic Geometry & 4.00 \\
2009 & TS: Introduction to Algebraic Geometry II & 4.00 \\
2009 & TS: De Rham Cohomology & 3.30 \\
2010 & TS: Hodge Theory & 3.25 \\
2013 & SS: Communicating Science & 4.00 \\
\end{performance}

\end{document}
